anitya/default_config.py

```diff
@@ -18,7 +18,8 @@
 SECRET_KEY = 'changeme please'
 
 # URL to the database
-DB_URL = 'sqlite:////var/tmp/anitya-dev.sqlite'
+repo_root = path.abspath(path.dirname(path.dirname(__file__)))
+DB_URL = 'sqlite:///{0}/anitya.db'.format(repo_root)
```

Comment:
apache should always have permission to write on ``/var/tmp``.

Since you figured out the way to override the DB_URL, I think I would prefer to keep the default as it is now.